### DISCUSSION TITLE
Doubt of Wysiwyg.

### DISCUSSION THREAD

**Discussion by Andres10121997** [twbs/bootstrap#33088] [General]
Hello.

I have a question, in Bootstrap is wysiwyg implemented? And if the answer is no, are they going to implement it in the future?

**Comment by mdo** [comment]
Nope, and nope :).

**Reply by Andres10121997** [reply]
Thanks for the reply.
Another question arose, why are they not going to implement it?
Since, in my point of view, it would be an innovation for Bootstrap, also, if some programmer or designer wanted to implement Wysiwyg in their program, using Bootstrap they could implement it.

Sorry if I have bad writing, English is not my first language.

**Reply by mdo** [reply]
It's a pretty big departure from what we do now. Just doesn't fit with what we want I accomplish with the project. Plus, there are some great options out there already.

**Reply by Andres10121997** [reply]
Ok, thanks.

**Reply by Andres10121997** [reply]
What other options do you recommend? Do you have another project, like Bootstrap or Bootstrap-icons, that I can use? to avoid compatibility problems.
